src/crypto/tls/tls_cf.go

```diff
@@ -1,3 +1,6 @@
+// Copyright 2021 Cloudflare, Inc. All rights reserved. Use of this source code
+// is governed by a BSD-style license that can be found in the LICENSE file.
+
```

Comment:
Which remains me that we should also update the other files that have this copyright to '2020-2021'

Reply:
TBH I have no idea what I'm doing with the copyright messages! I know how BSD license works when it's my own encode, but since we're extending someone else's work (i.e., the Go team's), I'm not sure if we get to hold copyright on this file or not.

Reply:
We should really define this ;) Maybe we can have a short call for the specific bureaucracy issues of cf-go?

Reply:
Yeah, though we can address this in a followup change.

Reply:
I'll bring it up in the t1000 meeting today.